### ISSUE TITLE
Rule `no-unsafe-spy` does not detect `jasmine.createSpyObj`

### ISSUE THREAD

**Issue by Valdermeyder** [tlvince/eslint-plugin-jasmine#112]
**Do you want to request a *feature* or report a *bug*?**
It is rather *bug* than *feature*

**What is the current behavior?**
If you declare outside of before/after/it block
```javascript
let someSpyObj = jasmine.createSpyObj('spyObj', ['spyMethod']);
```
warning will not be shown

**If the current behavior is a bug, please provide the steps to reproduce.**
Use `jasmine.createSpyObj` outside of before/after/it block

**What is the expected behavior?**
Warning should be shown when `jasmine.createSpyObj` is used outside of before/after/it block

**Please mention your node.js, eslint-plugin-jasmine and operating system version.**
node.js v7.2.1
eslint-plugin-jasmine v2.2.0

**Comment by DianaSuvorova** [comment]
Yes, it does not. Should be an easy fix. Will do.
